.travis.yml

```diff
@@ -50,7 +50,8 @@ install:
   - sudo ln -s -f /usr/bin/g++-7 /usr/local/bin/g++
   - mv src/Makefile.am src/Makefile_centos.am
   - mv src/Makefile_ubuntu.am src/Makefile.am
-  - .travis/check-git-clang-format-output.sh  
+  - clang-format -version
+  - .travis/check-git-clang-format-output.sh
```

Comment:
I don't think these tests are running on travis

Reply:
I've been having some issues with this, I think its fixed. I'll double check it soon.

Reply:
Fixed.